### PR TITLE
rimpy-pre-release: Add version 1.2.6.25

### PR DESCRIPTION
**Pull request by Zliced13**
Relates to #695

- [x] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/rimpy-pre-release.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "1.2.6.25",
+    "homepage": "https://steamcommunity.com/sharedfiles/filedetails/?id=1847679158",
+    "description": "Pre-release builds of the mod manager for the RimWorld game.",
```
Comment (Calinou) [inline bucket/rimpy-pre-release.json:4]:
```suggestion
    "description": "Pre-release builds of the mod manager for the RimWorld game",

```

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Comment by Zliced13** [comment]
> Thanks! Congratulations for your first merged pull request :tada:

Your welcome and thank you.

**Pull request merged**